app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -165,7 +168,7 @@ DonorsChooseDonationController.prototype.findProject = function(mobileNumber, co
         };
       }
       else {
-        sendSMS(mobileNumber, config.error_direct_user_to_restart);
```

Comment:
This `error_direct_user_to_restart ` property was removed in the 2015 schema and would have caused an error.